pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -522,7 +522,7 @@ func validateRestoreStatus(ar *admissionv1.AdmissionRequest, vm *v1.VirtualMachi
 	}
 
 	if !reflect.DeepEqual(oldVM.Spec, vm.Spec) {
-		if *vm.Spec.Running {
+		if (vm.Spec.Running != nil && *vm.Spec.Running) || (vm.Spec.RunStrategy != nil && *vm.Spec.RunStrategy != v1.RunStrategyHalted) {
```

Comment:
You can simplify this by `vm.RunStrategy()` and checking `!= v1.RunStrategyHalted`

Reply:
Changed in https://github.com/kubevirt/kubevirt/pull/6699/commits/c78a55f8b239b11ba3f0a33e91f689232dad1213